### PR TITLE
Add app.exception_traceback to context when an exception is thrown

### PR DESCRIPTION
**Pull request by kat-humu**
**What**
Add app.exception_traceback to context when an exception is thrown 

**Why?**
`app.exception_type` and `app.exception_string` give us insight into "what" exception occurred, adding `app.exception_traceback` gives more insight into "why" the exception occurred. It also keeps us from having to manually add context for every exception in our code.

### CODE REVIEW

**Review by MikeGoldsmith — APPROVED** [review]
Looks great, thanks @kat-humu!

**Comment by MikeGoldsmith** [comment]
@kat-humu we had an issue with a component of our CI blocking forked builds. Please could you merge main / rebase your changes to get the updated workflow. Thanks.

**Comment by kat-humu** [comment]
Done @MikeGoldsmith! Thanks.

**Comment by MikeGoldsmith** [comment]
Hi @kat-humu - thanks for rebasing your PR. To help us provide a consistent experience between beelines, I think we should maybe pick a key that isn't too specific for Python here. What do you think about `app.exception_stacktrace` - stacktrace is more general term that is used in many languages.

**Comment by kat-humu** [comment]
Great idea @MikeGoldsmith! Will make the change now.

File: beeline/trace.py [hunk 1]
```diff
@@ -73,6 +74,7 @@ def __call__(self, name, trace_id=None, parent_id=None):
                 span.add_context({
                     "app.exception_type": str(type(e)),
                     "app.exception_string": stringify_exception(e),
+                    "app.exception_traceback": traceback.format_exc(),
```
Comment (kat-humu) [inline beeline/trace.py:77]:
```suggestion
                    "app.exception_stacktrace": traceback.format_exc(),
```

**Comment by MikeGoldsmith** [comment]
Great, thanks @kat-humu 😄

**Pull request merged**